ci-operator/step-registry/aks/provision/aks-provision-commands.sh

```diff
@@ -26,18 +31,15 @@ if [[ "${AKS_CLUSTER_AUTOSCALER_MAX_NODES:-}" != "" ]]; then
     CLUSTER_AUTOSCALER_ARGS+=" --max-count ${AKS_CLUSTER_AUTOSCALER_MAX_NODES}"
 fi
 
-az --version
-az login --service-principal -u "${AZURE_AUTH_CLIENT_ID}" -p "${AZURE_AUTH_CLIENT_SECRET}" --tenant "${AZURE_AUTH_TENANT_ID}" --output none
-
 echo "Creating resource group for the aks cluster"
 RESOURCEGROUP="${RESOURCE_NAME_PREFIX}-aks-rg"
 az group create --name "$RESOURCEGROUP" --location "$AZURE_LOCATION"
 echo "$RESOURCEGROUP" > "${SHARED_DIR}/resourcegroup_aks"
 
 K8S_VERSION_ARGS=""
 if [[ "${USE_LATEST_K8S_VERSION:-}" == "true" ]]; then
-  K8S_LATEST_VERSION=$(az aks get-versions --location "${AZURE_LOCATION}" --output json --query 'max(orchestrators[*].orchestratorVersion)')
-  K8S_VERSION_ARGS="--kubernetes-version ${K8S_LATEST_VERSION}"
+    K8S_LATEST_VERSION=$(az aks get-versions --location "${AZURE_LOCATION}" --output json --query 'max(orchestrators[*].orchestratorVersion)')
```

Comment:
how about setting it configurable in the step? I mean it would not be the latest one in the aro hcp service in the future. So we could rehearsal one special version of aks CI quickly to debug some online issues if needed.

Reply:
Updated